phpBB/config/default/container/services_event.yml

```diff
@@ -9,6 +9,7 @@ services:
         arguments:
             - '@template'
             - '@language'
+            - '%?debug.exceptions'
```

Comment:
what is this notation?

Reply:
I used this because I thought `debug.exceptions` would not be available all the time, so to avoid errors when compiling the container I used this notation which should say that the argument is optional (which it is, basically)

Reply:
I think you forgot a % at the end and it will always be defined, no matter what

Reply:
Oh, you are correct.